go/libkb/keyfamily.go

```diff
@@ -315,40 +315,15 @@ func (cki ComputedKeyInfos) InsertServerEldestKey(eldestKey GenericKey, un Norma
 
 func (ckf ComputedKeyFamily) InsertEldestLink(tcl TypedChainLink, username NormalizedUsername) (err error) {
 	kid := tcl.GetKID()
-	key, err := ckf.FindKeyWithKIDUnsafe(kid)
+	_, err = ckf.FindKeyWithKIDUnsafe(kid)
 	if err != nil {
 		return
 	}
```

Comment:
We used to use the key below, so if we ever introduced a bug that someone returned nil here without an error, it would've caused a panic. Now it might silently pass through here. It's a super minor thing; I definitely don't feel strongly about it.